bin/pic-submit

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+#
+# Copyright 2013-2018 Axel Huebl, Rene Widera, Richard Pausch
```

Comment:
@psychocoderHPC just add yourself :)

Reply:
@ax3l I am `Rene Widera`^^

Reply:
Yeah, I mean *exclusively* you ;-) but ok if it's copied a lot...